provider/lib/utils.js

```diff
@@ -98,6 +98,9 @@ module.exports = function(
         var method = 'postTrigger';
 
         return new Promise(function(resolve, reject) {
+            var triggerIdentifier = utils.getTriggerIdentifier(dataTrigger.apikey, dataTrigger.namespace, dataTrigger.name);
```

Comment:
this will give you the same result with only a one line change
https://github.com/jasonpet/openwhisk-package-alarms/commit/33705b8c8314fe13fc16b8bd36ad85d200e3a4c6